docs/current/cli/465058-install.md

```diff
@@ -0,0 +1,3 @@
+---
+slug: /cli/install
```

Comment:
@marcosnils 
- are we providing binaries?
- what will be the process to install for windows/linux/macos?
- are there any specific prerequisites to install/use the binaries?

Reply:
> * are we providing binaries?

yes!


> * what will be the process to install for windows/linux/macos?

Same as the previous `dagger` CLI from the Europa docs




> * are there any specific prerequisites to install/use the binaries?

not really

Reply:
Have added these steps now, please check